include/powsybl/iidm/HvdcLine.hpp

```diff
@@ -22,7 +21,7 @@ namespace iidm {
 class HvdcConverterStation;
 class Network;
 
-class HvdcLine : public Identifiable, public MultiVariantObject {
+class HvdcLine : public Identifiable {
```

Comment:
You forget to change the implementation of the MultiVariantObject methods to call the identifiable code. MultiVariant extensions of HvdcLine will not be notified by a variant change

Reply:
Ok, done by backporting Java PR #1476